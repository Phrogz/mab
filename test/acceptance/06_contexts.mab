p( "Test#6" )
setSlot( 'go', method(
	setSlot( 'gadget', method(
		p( self )
		p( context self )
		p( context )
		p( context self self )
	))
	gadget
))
go