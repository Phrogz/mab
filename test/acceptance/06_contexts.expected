Test#6
<Context of 'go'>
<Context of 'go'>
<Context of 'gadget'>
Lawn